src/browser/stacktrace/constants.ts

```diff
@@ -6,3 +6,7 @@ export const WDIO_IGNORED_STACK_FUNCTIONS = [
     "Element.newCommand",
     "Element.elementErrorHandlerCallbackFn",
 ];
+// Other frames are being filtered out by error-stack-parser
+// Declared at https://github.com/stacktracejs/error-stack-parser/blob/v2.1.4/error-stack-parser.js#L17
+// Used at https://github.com/stacktracejs/error-stack-parser/blob/v2.1.4/error-stack-parser.js#L52-L54
+export const STACK_FRAME_REG_EXP = /^\s*at .*(\S+:\d+|\(native\))/m;
```

Comment:
Example - 
`    at Reflect.get (<anonymous>)`
is a valid frame, but it is not supported by error-stack-parser, though
`    at Object.proxyGetter [as get] (...)`
which is nearby, is supported.
By filtering these in `filterExtraStackFrames` we ensure [rawFramesArr.length === framesParsed.length](https://github.com/gemini-testing/testplane/pull/1008/files#diff-f659cdb6ee50d63a229978b33f5d3d2c982c3acff969c479a49416a1201b8c9aR157) (previously we could see missmatch in frames length and refuse to do anything in order to not make it worse
Since those are useless frames (for example, you will still be able `Object.proxyGetter [as get]` with filepath nearby), its better to remove them, so we can filter out other extra stack frames consistently